docs/docs/browser-support.md

```diff
@@ -1,7 +1,22 @@
 ---
-title: Browserslist
+title: Browser Support
 ---
 
```

Comment:
Add/replace the text at the start saying "Gatsby supports [the same browsers as the stable version of React.js](https://facebook.github.io/react/docs/react-dom.html#browser-support) which is currently IE9+ as well as the most recent versions of other popular browsers. Gatsby requires support for the Promise API which is not supported in older browsers. Because of this, Gatsby…"